schemas/documents/registry/creditClassPage.js

```diff
@@ -15,5 +15,11 @@ export default {
       type: 'image',
       validation: Rule => Rule.required(),
     },
+    {
+      title: 'Book a call link',
```

Comment:
in the app, we were only using `buyModalOptions` data to get the "book a call" button link on a credit class page but not the rest of the data anymore so deleted `buyModalOptions` and added a simple link to `creditClassPage`